packages/kit/src/resolve.ts

```diff
@@ -151,7 +151,7 @@ async function isDirectory (path: string) {
   return (await fsp.lstat(path)).isDirectory()
 }
 
-export async function resolveFiles (path: string, pattern: string | string[]) {
-  const files = await globby(pattern, { cwd: path, followSymbolicLinks: true })
+export async function resolveFiles (path: string, pattern: string | string[], opts?: { followSymbolicLinks?: boolean }) {
+  const files = await globby(pattern, { cwd: path, followSymbolicLinks: opts?.followSymbolicLinks || true })
```

Comment:
when `followSymbolicLinks` is `false`, it will evaluate to `false || true` (true). 
```suggestion
  const files = await globby(pattern, { cwd: path, followSymbolicLinks: opts?.followSymbolicLinks ?? true })
```